tests/app/test_settings.py

```diff
@@ -14,17 +14,19 @@
 class TestSettings:
     def test_init_path(self):
         # Test 1 all defined:
-        settings = Settings(library_dir="/home/user/library", data_dir="/home/user/data", model_dir="/home/user/model")
+        settings = Settings(
+            library_dir="/home/user/library", media_dir="/home/user/media", models_dir="/home/user/model"
+        )
 
-        assert settings.library_dir == "/home/user/library"
-        assert settings.data_dir == Path("/home/user/data")
-        assert settings.model_dir == Path("/home/user/model")
+        assert settings.library_dir == Path("/home/user/library")
+        assert settings.media_dir == Path("/home/user/media")
+        assert settings.models_dir == Path("/home/user/model")
 
         # Test 2 all undefined:
         settings = Settings()
-        assert settings.library_dir == (Path.cwd() / "library").as_posix()
-        assert settings.data_dir == Path(settings.library_dir)
-        assert settings.model_dir == settings.data_dir / "models"
+        assert settings.library_dir == Path.cwd() / "library"
+        assert settings.media_dir == Path.cwd() / "media"
+        assert settings.models_dir == settings.library_dir / "models"
```

Comment:
before, models_dir has been set to /home/dir/model (without 's'). Choose "model" or "models" with consistency